packages/vscode-extension/PRERELEASE.md

```diff
@@ -4,15 +4,36 @@
 
 > Note: This changelog only includes the changes for the pre-release versions of Teams Toolkit. For the changelog of stable versions, please refer to the [Teams Toolkit Changelog](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/CHANGELOG.md).
 
-### April 01, 2024
-
-#### New Features
-
-- **Word, Excel and PowerPoint Add-ins in Teams Toolkit**
-![WXP Add-in](https://github.com/OfficeDev/TeamsFx/assets/11220663/30679a8c-b0b0-4b1c-ad4f-114547a12a6b)
-Teams Toolkit now supports Microsoft Word, Excel, or PowerPoint JavaScript add-in development. This support enables developers to quickly get started and build add-ins with high productivity, featuring:
-  - Code samples for various add-in types such as task pane, content, or ribbon. Developers can customize these samples to create their own add-in projects for Word, Excel and PowerPoints.
-  - A side pane offering a unified and centralized experience for checking dependencies, running and debugging add-ins, managing lifecycle, leveraging utility, getting help, and providing feedback.
+### April 15, 2024
+
+#### New Features 
+
+- **Create API based Message Extensions using auth-protected API** </br> 
+  Teams Toolkit supports two types of API authentication protection in your API based Message Extension app: </br> 
+  ![add-auth-api-me](https://github.com/OfficeDev/TeamsFx/assets/113089977/c5faea2f-676b-4a8c-82d6-f3b037e54f0e) 
+  - API-Key: you can either add the API key of your existing API, or if you don't have an API, Teams Toolkit will generate one to show how authentication works. 
+  - Microsoft Entra (Azure AD): Teams Toolkit can help you create Microsoft Entra ID to authenticate your new API. 
+
+- **Debug Message Extensions in Teams App Test Tool** </br> 
+  Teams App Test Tool helps developers to debug and test in a web-based environment that emulates Microsoft Teams without using tunnels or Microsoft 365 account. In this version we add Teams App Test Tool support to all types of Message Extension app, including search-based, action-based and link unfurling Message Extension app. 
```

Comment:
API-based Message Extension app is not supported yet in Teams App Test Tool, shall we refine below sentence to avoid confusion?
> In this version we add Teams App Test Tool support to all types of Message Extension app,

Reply:
updated